.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
No I was saying that,  Photologue searches for media images in media folder. And by removing that it may lead to errors. Have you tested the code for errors while retrieving media from carousel and tag app. There is only way to access those media paths from media folder only (currently).